duckduckgo/BrowserServicesKit
# WebCompat changes

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL:  https://app.asana.com/0/1200223097357040/1202137014114084/f
iOS PR:  
macOS PR: https://github.com/more-duckduckgo-org/macos-browser/pull/547

**Optional**:

Tech Design URL:
CC:

**Description**:

1. This moves "navigatorCredentials" to "webCompat" so we can in future fix breakage with a single feature.
2. This imports in https://github.com/duckduckgo/content-scope-scripts/pull/52 which fixes the outerHeight/width sizing issue.

**Steps to test this PR**:
1. Go to https://myaccount.payoneer.com/
2. Should redirect to: https://login.payoneer.com/

In stable this page isn't redirecting as window.outerWidth and height are 0.



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
